deployment/Puppetfile

```diff
@@ -1,5 +1,6 @@
 forge "https://forgeapi.puppetlabs.com"
 
+mod 'garethr-docker', '5.3.0'
```

Comment:
Why not use the [puppetlabs/docker_platform](https://forge.puppet.com/puppetlabs/docker_platform)?